### PR TITLE
Frontend Integration and Function Enhancements

### PR DESCRIPTION
**Pull request by Anshu370**
~Frontend completed and successfully connected with script.py.
~Modified the encrypt_file and decrypt_file functions to handle both GUI and CLI functionality for smoother user experience.
~Ensured compatibility between the two interfaces.
->Requirements
    ~pip install customtkinter
Images 
Screenshot 2024-10-18 190051

### CODE REVIEW

File: Frontend/main.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import customtkinter as ctk
-
+import script
```
Comment (Vikranth3140) [inline Frontend/main.py:2]: Please use the below line before trying to import script from the parent directory as `ModuleNotFoundError` will occur
```
sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
```

File: Frontend/main.py [hunk 2]
```diff
@@ -19,33 +19,45 @@ def CenterWindowToDisplay(Screen: ctk, width: int, height: int, scale_factor: fl
...
-key_frame = ctk.CTkFrame(app)
-key_frame.pack(anchor=ctk.N, fill=ctk.X)
-
-def Ebrowse():
+def Ebrowsekey():
 
     filename = ctk.filedialog.askopenfilename(filetypes=[("Encryption", "*.txt*"), ("Decryption", "*.decrypted*")])
     encryptfilename.insert(0, filename)
-def Dbrowse():
+def Dbrowsekey():
```
Comment (Vikranth3140) [inline Frontend/main.py:26]: Please follow better code formatting and practices.
You can refer to https://peps.python.org/pep-0008/ for the same.

File: Frontend/main.py (hunk 1 above)
Comment (Vikranth3140) [inline Frontend/main.py:2]: Please use the below line before trying to import script from the parent directory as `ModuleNotFoundError` will occur
```
sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
```

**Comment by Vikranth3140** [comment]
Hey @Anshu370,
Thanks for sending a PR for this #7 issue.

Please check the reviews i have left and also please test the application and send me a clip if possible.
I do not think the password matching is robust and correctly implemented as of now.

**Review by Anshu370 — COMMENTED** [review]
Okay I will do the following changes.

File: Frontend/main.py (hunk 2 above)
Comment (Anshu370) [reply Frontend/main.py:26]: okay

**Comment by Anshu370** [comment]
Hey @Vikranth3140, 
This PR for #7 is completed.

I had fixed all the issues and test them here is the clip for the same.
I have recorder both the Scenario  before and after of the code correction.

https://github.com/user-attachments/assets/a46810b1-69eb-4233-a4f6-902f451cb66e

**Review by Vikranth3140 — CHANGES_REQUESTED** [review]
Please enhance with the below:

1. Increase the size of the password dialogue box.
2. Make sure the `secret.key` is initialized in the root directory and in the `FrontEnd` directory.

**Comment by Vikranth3140** [comment]
Hey @Anshu370,

Thanks for the changes, I have reviewed the changes and it looks good to go.

A couple of changes requested for enhancement.

1. Increase the size of the password dialogue box.
2. Make sure the `secret.key` is initialized in the root directory and in the `FrontEnd` directory.
3. Sync the changes back to the original repo to resolve merge conflicts.

After this we are good to go for your First Accepted PR!

**Comment by Anshu370** [comment]
okay

**Pull request closed without merging**